go/src/koding/socialapi/transport_test.go

```diff
@@ -0,0 +1,142 @@
+package socialapi_test
+
+import (
+	"encoding/json"
+	"log"
+	"net"
+	"net/http"
+	"strings"
+	"testing"
+
+	"koding/kites/tunnelproxy/discover/discovertest"
+	"koding/socialapi"
+)
+
+func TestTransport(t *testing.T) {
+	var storage TrxStorage
```

Comment:
i believe you can. it is same implementation with an inmemory storage with an GetAll(maybe?) function, and it would eliminate some of the code